src/apis/ConfigApis.js

```diff
@@ -1,20 +1,24 @@
 import axios from 'axios'
-
 import { loadProgressBar } from 'axios-progress-bar'
 
-if (process.env.NODE_ENV === 'development') {
-  axios.defaults.socketUrl = location.protocol + '//' + location.hostname + ':' + process.env.PORT
-  axios.defaults.baseURL = axios.defaults.socketUrl + '/api'
-} else {
-  var port = parseInt(window.location.port) || (window.location.protocol === 'https:' ? 443 : 80)
-  axios.defaults.socketUrl = location.protocol + '//' + location.hostname + ':' + port
-  axios.defaults.baseURL = '/api'
+function getBasePath () {
+  return document.baseURI.replace(/\/$/, '')
 }
 
+axios.defaults.socketUrl = getBasePath()
+axios.defaults.baseURL = axios.defaults.socketUrl + '/api'
```

Comment:
since you've not concatinated strings like that elsewhere
```suggestion
axios.defaults.baseURL = `${axios.defaults.socketUrl}/api`
```